### PR TITLE
Commerce 11107

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-11107

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java [hunk 1]
```diff
@@ -902,9 +909,20 @@ private CPDefinition _updateNestedResources(
...
 		if (skus != null) {
 			for (Sku sku : skus) {
+				Map<String, Serializable> expandoBridgeAttributes =
+					_getExpandoBridgeAttributes(sku);
+
+				serviceContext.setExpandoBridgeAttributes(
+					expandoBridgeAttributes);
+
+				serviceContext.setExpandoBridgeAttributes(
+					expandoBridgeAttributes);
```
Comment (alessiorendina) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java:919]: Why is it done 2 times?

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@andrea-ale-sbarra please check my comment, thanks!

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java (hunk 1 above)
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java]: mistake

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce

**Review by alessiorendina — APPROVED** [review]

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Pull request closed without merging**